python/lsst/ts/mtmount/mtmount_csc.py

```diff
@@ -806,7 +816,7 @@ async def enable_devices(self):
                 commands.OilSupplySystemPower(on=True),
                 # Cannot successfully reset the axes alarms
                 # until the main power supply is on.
-                # Sometimes a second reset is needed for the main axes
+                # Sometimes a second reset is needed for the both axes
```

Comment:
` for the both axes` -> `for both axes`?

Reply:
Thanks. Changing that comment was an accident. I reverted the change.